arch/arm64/boot/dts/rockchip/overlays/rk3588-disable-sdhci.dts

```diff
@@ -0,0 +1,16 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	metadata {
+		title = "Disable SDHCI";
+		compatible = "radxa,rock-5-itx";
+		category = "misc";
+		exclusive = "sdhci";
+		description = "Disable SDHCI.";
```

Comment:
```suggestion
		description = "Disable SDHCI.
This will disable access to eMMC.
If your system is installed on the eMMC, it will now be unbootable!";
```

Reply:
格式改了，之前有的产品描述太多写同一行太拥挤，就直接换行了：https://github.com/radxa-pkg/radxa-overlays/blob/main/arch/arm64/boot/dts/rockchip/overlays/rk3568-fiq-debugger-uart0.dts

Reply:
OK，改过来了